### ISSUE TITLE
RSpec 'expect' syntax not working

### ISSUE THREAD

**Issue by vjramth** [inspec/inspec#3617]
Recently we did a Inspec software upgrade from Inspec-2.1.43 to latest Inspec-3.0.46 and we are facing a issue with Expect method whose tests worked fine while running from inspec-2.1.43, but throws error saying undefined method 'expect'

```
describe 'XXXX' do
  it 'has the required client queue(s) running' do
    **expect**($results.compact.count).to eq 1
  end
end
```

```
describe 'XXXX' do
  it 'has the required port(s) listening' do
    **expect**($portresults.compact.count).to eq 1
  end
end
```

**Error:**
```
XXXX queue
 ×  has the required client queue(s) running
     **undefined method `expect'** for #<RSpec::ExampleGroups::XXXXQueue:0x000000000abcdsed>

 XXXX Port
     ×  has the required port(s) listening
     **undefined method `expect'** for #<RSpec::ExampleGroups::XXXXPort:0x00000000012345as>

```

Works fine when run from Inspec-2.1.43
```
XXXX queue
     ✔  has the required client queue(s) running
XXXX Port
     ✔  has the required port(s) listening
```

EDIT (@clintoncwolfe): place code in blocks

**Comment by clintoncwolfe** [comment]
Likely related to #3560, which sets the RSpec preferred syntax to `should` mode.  That was shipped in 3.0.46.  The prior RubyGems release was 3.0.25 - try that

**Comment by clintoncwolfe** [comment]
Also reported on #3616, closing this as dupe
